CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ First fork and clone the repository. If you're not sure how to do this, please w
 Run:
 
 ```bash
-npm install
+npm install && npm run site:build
```

Comment:
We just add a discussion with @erezrokah and were thinking `npm run site:build:install` might actually make more sense, since the linting errors are only due to uninstalled dependencies. What do you think?

Reply:
That makes sense 👍  I'll make the update

Originally I was hesitant to use `npm ci` when reading into `site:build:install`. Now I think it's a reasonable decision over `npm install`, since it'll stay out of the way, installing stable dependencies. I'm assuming there's more value to smoothing the starting path than staying compatibility with the state of the user's `site/node_modules`, since I'm guessing they're coming to the contributing getting started as a landing page or while debugging. Also a vast majority of contributions are probably made to the main project, so I figure it's a non-issue and changes made to `site` will be done by someone familiar with the setup.